adapters/persistence/adapter-maria/src/main/java/com/caregiver/domain/Group.java

```diff
@@ -0,0 +1,48 @@
+package com.caregiver.domain;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Builder
+public class Group {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "group_id")
+    private long id;
+
+    @Column(name = "group_name", nullable = false)
+    private String groupName;
+
+    @Column(name = "minimum_capacity", nullable = false)
+    private Integer minimumCapacity;
```

Comment:
```suggestion
    private int minimumCapacity;
```
`nullable` 이 `false` 라면 `int` 로 해도 되지 않을까요???

Reply:
minimumCapacity 값이 무조건 null 이 아니기 때문에 int 여도 된다고 생각하신건가요~? 🤔 
저는 @Column 의 nullable 은 테이블의 "defalut null 이 가능하다" 를 의미한다고 생각해서 이러한 의미로 nullable = false 를 넣었습니다.

Reply:
`null` 이 가능하다라면 `nullable = true` 가 되어야 하지 않을까요? 

그리고 RDB 에서 `null` 을 사용하는 것은 가능하다면 지양하는게 좋을 거같습니다. `default` 값을 지정하거나요.

`null` 을 사용한다는것은 알수 없는 경우를 만드는 것이고 DB 성능에도 영향을 미칩니다.

애플리케이션에서도 기본형 타입이 성능면에서도 이점이 있습니다.

- 이펙티브 자바 아이템 61
![image](https://user-images.githubusercontent.com/65605968/118240467-02ecb380-b4d6-11eb-8662-c29f47dc811b.png)

Reply:
제가 반대로 이야기 했군요;; 
Integer 타입일 때, nullable = false 이면 @ClumnDefalut("0") 같이 처리 할 수 있겠으나 그럴 필요가 딱히 없겠네요.

말씀하신 대로 int 로 가는게 좋겠습니다 👍